src/DefaultTemplate.tsx

```diff
@@ -42,5 +42,6 @@ export const DefaultTemplate: React.FunctionComponent<TemplateProps<any>> = prop
 export const defaultTemplate: Template<any> = {
   id: "default-template",
   label: "Default",
+  type: "custom-template",
```

Comment:
follow up of what we discussed about, this shows that the API changed, and thus that the PR introduce a breaking change